npm/cmd/start.go

```diff
@@ -115,8 +117,20 @@ func start(config npmconfig.Config, flags npmconfig.Flags) error {
 	factor := rand.Float64() + 1 //nolint
 	resyncPeriod := time.Duration(float64(minResyncPeriod.Nanoseconds()) * factor)
 	klog.Infof("Resync period for NPM pod is set to %d.", int(resyncPeriod/time.Minute))
-	factory := informers.NewSharedInformerFactory(clientset, resyncPeriod)
 
+	factory := informers.NewSharedInformerFactory(clientset, resyncPeriod)
+	// npm-lite -> daemon set will listen to pods only in its own node
+	if config.Toggles.EnableNPMLite {
+		nodeName := os.Getenv("NODE_NAME")
```

Comment:
nit: I think we get the node name down below. Could probably share the same variable

Reply:
yes changed it, thanks